internal/cmd/db_shell.go

```diff
@@ -188,224 +125,19 @@ func tokenFromDb(db *turso.Database, client *turso.Client) (string, error) {
-	Message string
-}
-
-func (e *SqlError) Error() string {
-	return e.Message
-}
-
-func query(url, token, stmt string) error {
-	switch stmt {
-	case ".tables":
-		stmt = getTables()
-	case ".schema":
-		stmt = getSchema()
-	}
-	resp, err := doQuery(url, token, stmt)
-	if err != nil {
-		return err
-	}
-	defer resp.Body.Close()
-	body, err := io.ReadAll(resp.Body)
-	if err != nil {
-		return err
-	}
-	if resp.StatusCode != http.StatusOK {
-		var err_response ErrorResponse
-		if err := json.Unmarshal(body, &err_response); err != nil {
-			return &SqlError{fmt.Sprintf("Failed to execute SQL statement: %s\n%s", stmt, err)}
-		}
-		var msg string
-		if err_response.Message == "interactive transaction not allowed in HTTP queries" {
-			msg = "Transactions are only supported in the shell using semicolons to separate each statement.\nFor example: \"BEGIN; [your SQL statements]; END\""
-		} else {
-			msg = fmt.Sprintf("Failed to execute SQL statement: %s\n%s", stmt, err_response.Message)
-		}
-		return &SqlError{msg}
-	}
-
-	var results []QueryResult
-	if err := json.Unmarshal(body, &results); err != nil {
-		return err
-	}
-	errs := []string{}
-	for _, result := range results {
-		if result.Error != nil {
-			errs = append(errs, result.Error.Message)
-		}
-		if result.Results != nil {
-			columns := make([]interface{}, 0)
-			for _, column := range result.Results.Columns {
-				columns = append(columns, strings.ToUpper(column))
-			}
-			tbl := table.New(columns...)
-			for _, row := range result.Results.Rows {
-				for idx, v := range row {
-					if f64, ok := v.(float64); ok {
-						row[idx] = strconv.FormatFloat(f64, 'f', -1, 64)
-					} else if v == nil {
-						row[idx] = "NULL"
-					} else if m, ok := v.(map[string]interface{}); ok {
-						if value, ok := m["base64"]; ok {
-							if base64Value, ok := value.(string); ok {
-								bytes := make([]byte, base64.StdEncoding.DecodedLen(len(base64Value)))
-								count, err := base64.StdEncoding.Decode(bytes, []byte(base64Value))
-								if err != nil {
-									row[idx] = base64Value
-								} else {
-									row[idx] = hex.EncodeToString(bytes[:count])
-								}
-							}
-						}
-					}
-				}
-				tbl.AddRow(row...)
-			}
-			tbl.Print()
-		}
-	}
-	if len(errs) > 0 {
-		return &SqlError{(strings.Join(errs, "; "))}
-	}
-
-	return nil
-}
-
-func doQuery(url, token, stmt string) (*http.Response, error) {
-	stmts, err := sqlparser.SplitStatementToPieces(stmt)
-	if err != nil {
-		return nil, err
-	}
-	rawReq := QueryRequest{
-		Statements: stmts,
-	}
-	body, err := json.Marshal(rawReq)
-	if err != nil {
-		return nil, err
-	}
-	req, err := http.NewRequest("POST", url, bytes.NewReader(body))
-	if err != nil {
-		return nil, err
-	}
-	if token != "" {
-		req.Header.Add("Authorization", fmt.Sprintf("Bearer %s", token))
-	}
-	return http.DefaultClient.Do(req)
+func addTokenAsQueryParameter(dbUrl string, token string) string {
+	return fmt.Sprintf("%s/?jwt=%s", dbUrl, token)
```

Comment:
Do we need `/?`? I thought `/` is just needed in zsh to escape `?` but here it shouldn't be needed, no?

Reply:
It's not zsh escape, it's just a optional final slash in the end of a path that I'm used to add, but it's unnecessary